core/templates/components/entity-creation-services/skill-creation.service.ts

```diff
@@ -31,8 +35,25 @@ angular.module('oppia').factory('SkillCreationService', [
     var CREATE_NEW_SKILL_URL_TEMPLATE = (
       '/skill_editor/<skill_id>');
     var skillCreationInProgress = false;
+    var skillDescriptionStatusMarker = 1;
 
     return {
+      markChangeInSkillDescription: function() {
+        skillDescriptionStatusMarker *= -1;
```

Comment:
I have no idea what -1, 0, 1 mean ... I suggest you use enums, i.e. constants that are of the form SKILL_DESCRIPTION_STATUS_* instead.

Reply:
Done, can you see if the naming is fine?